packages/flutter/lib/src/material/icon_theme_data.dart

```diff
@@ -2,15 +2,22 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:ui' as ui;
+
 enum IconThemeColor { white, black }
 
 class IconThemeData {
-  const IconThemeData({ this.color });
+  const IconThemeData({ this.color, this.alpha });
+
   final IconThemeColor color;
+  final double alpha;
```

Comment:
Aha, I wasn't aware of the convention.  That's helpful. I've changed all of "alphas" that were really 0.0-1.0 opacities to "opacity".